website/docs/aggregation operations/aggregation-pipeline-and-commands.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 1
+slug: /aggregation-pipeline-and-commands/
+---
+
+# Aggregation pipeline and commands
+
+Aggregation operations involve performing various operations on a large number of data records, such as data grouping, sorting, restructuring, or modifying.
+These operations pass through one or more stages, which make up a pipeline.
+
+![aggregation stages](../../static/img/docs/aggregation-stages.jpg)
+
+Each stage acts upon the returned documents of the previous stage, starting with the input documents.
+As shown above, the documents pass through the pipeline with the result of the previous stage acting as input for the next stage, going from `$count` => `$group` => `$sort` stage.
```

Comment:
Let's maybe come up with an example to demonstrate how it works? I like our docs where we show the exact examples of documents and how they are modified.